javascript/javascript2/week1/shortestWord.js

```diff
@@ -0,0 +1,12 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium", "å"];
+
+function shortestWord(wordList) {
+  let shortWord = wordList[0];
+  for (let i = 1; i < wordList.length; i++) {
+    if (wordList[i].length < shortWord.length) {
+      shortWord = wordList[i];
+    }
+  }
+  return shortWord;
+}
+console.log(shortestWord(danishWords));
```

Comment:
If you see the array, the shortest word consist only one letter?  can you put that condition as well?

There are two words with single letter, in that case which one will be returned as shorted? should be returning both right? please check the logic again.

Reply:
implemented this

Reply:
Good